graphwalker-dsl/src/main/java/org/graphwalker/dsl/antlr/dot/AntlrDotListener.java

```diff
@@ -44,8 +44,8 @@
 public class AntlrDotListener extends DOTBaseListener {
     private static final Logger logger = LoggerFactory.getLogger(AntlrDotListener.class);
 
-    public Map<String, Vertex> vertices = new HashMap<>();
```

Comment:
The attributes vertices and edges are accessed outside the class, so you need to add and implement getters in order for this to compile.